resources/building-mobile-extensions/building-an-extension.md

```diff
@@ -49,6 +49,7 @@ All Adobe module names are prefixed with `com.adobe.module` and are considered r
 
 The `Extension` class has the following methods that you can optionally override and a member that provides access to the Event Hub:
 
+* `getFriendlyName`: Retuns a friendly name for your extension.
```

Comment:
Just to confirm, will your average user know what "a friendly name" entails?

Reply:
I guess so, @emdobrin do you think we should provide more details about what the friendly name is and how we're using it?

Reply:
I think this should be sufficient. We can add something like: `Returns a friendly name for your extension, usually referenced in logs.`